src/main/java/dao/AirportDao.java

```diff
@@ -0,0 +1,20 @@
+package dao;
+
+import entities.Airport;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Set;
+
+/**
+ * DAO interface for airport entity
+ * @author klysov
+ */
+public interface AirportDao {
+    Airport extractAirportFromResultSet(ResultSet rs) throws SQLException;
+    Set<Airport> getAllAirports();
+    Airport getAirportById(int id);
+    boolean insertAirport(Airport airport);
+    boolean updateAirport(Airport airport);
+    boolean deleteAirport(Airport airport);
+}
```

Comment:
when you will call all of this methods it will be like airportDao.getAirportById(someInt) so you can name it just getById and it can help you to create some super interface like Dao<T>. the same think for all other dao interfaces and implementations